lib/param_parsing/versiontf.go

```diff
@@ -0,0 +1,26 @@
+package param_parsing
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-config-inspect/tfconfig"
+	"github.com/warrensbox/terraform-switcher/lib"
+	"log"
+)
+
+func GetVersionFromVersionsTF(params Params) Params {
+	filePath := params.ChDirPath + "/version.tf"
```

Comment:
I think sticking with quite specific file name isn't correct approach as TF version constraint can be put into any `.tf` file. Instead we need to assign path to the directory here.